instructor/utils.py

```diff
@@ -121,10 +122,12 @@ async def extract_json_from_stream_async(
 
 def update_total_usage(
     response: T_Model,
-    total_usage: OpenAIUsage | AnthropicUsage,
+    total_usage: CompletionUsage | AnthropicUsage,
 ) -> T_Model | ChatCompletion:
     response_usage = getattr(response, "usage", None)
-    if isinstance(response_usage, OpenAIUsage) and isinstance(total_usage, OpenAIUsage):
+    if isinstance(response_usage, (CompletionUsage, LiteLLMUsage)) and isinstance(
```

Comment:
this will make litellm a dependaence, is there a prototype or inferface we can use isntead